docs/integrations.md

```diff
@@ -0,0 +1,26 @@
+# Integrations
+
+## Weights & Biases
+
+LLM Engine integrates with Weights & Biases to track metrics during fine tuning. To enable:
+
+```python
+from llmengine import FineTune
+
+response = FineTune.create(
+    model="llama-2-7b",
+    training_file="s3://my-bucket/path/to/training-file.csv",
+    validation_file="s3://my-bucket/path/to/validation-file.csv",
+    hyperparameters={"report_to": "wandb"},
+    wandb_config={"api_key":"key", "project":"fine-tune project"}
+)
+```
+
+Configs to specify:
+
+| Field             | Subfield                                                             | Note                                                      |
+| ----------------- | -------------------------------------------------------------------- | --------------------------------------------------------- |
+| `hyperparameters` | `report_to`                                                          | Set to `wandb` to enables automatic metrics tracking      |
+| `wandb_config`    | `api_key`                                                            | The API key, must specify                                 |
```

Comment:
Might as well adopt a convention of `(Optional)` or `(Required)` for each, instead of "must specify".